lib/rialto/etl/service_client/entity_resolver.rb

```diff
@@ -16,10 +16,15 @@ def self.resolve(type, params)
         end
 
         def initialize
-          @conn = connection
+          initialize_connection
```

Comment:
Can you say a bit about this change? Was it to facilitate testing (allowing you to sub in dummy values)?

Reply:
Yes, it was for testing. Otherwise, the connection values might come from local settings, which break the test.